redhat-cop/containers-quickstarts
# add github workflows for gitlab-runners/ubi7-gitlab-runner and update github workflow for gitlab-runners/ubi8-git

#### What is this PR About?
* add github workflows for gitlab-runners/ubi7-gitlab-runner
* update github workflow for gitlab-runners/ubi8-git

#### How do we test this?
The workflow will run.

You can see example runs against my own fork:
https://github.com/itewk/containers-quickstarts/actions

cc: @redhat-cop/day-in-the-life
